js/screens/ActivityDetails/ActivityDetailsContainer.js

```diff
@@ -17,9 +17,8 @@ export default class ActivityContainer extends Component {
     headerStyle: {
       ...globalStyles.navigationHeader
     }
-  };
-  toggleVisibility = () => {
-    this.setState({ isVisible: !this.state.isVisible });
+  // toggleVisibility = () => {
+    // this.setState({ isVisible: !this.state.isVisible });
```

Comment:
remove all the unnecessary comments

Reply:
einer's file -- i just commented it out so it wouldn't break my code.